operator/v1/types_console.go

```diff
@@ -146,6 +146,13 @@ type ConsoleCustomization struct {
 	// +listMapKey=id
 	// +optional
 	Perspectives []Perspective `json:"perspectives"`
+	// consoleURL is an URL to be used as the base console address.
+	// If not specified, the console route hostname will be used.
+	// This is required when using console on clusters where the ingress capability is disabled.
```

Comment:
nit:
I would add additional warning which would make it clear to add a resolvable URL.
```
// Invalid value will prevent a console rollout.
```

Reply:
Done.